packages/marshal/src/encodeToSmallcaps.js

```diff
@@ -175,37 +189,30 @@ export const makeEncodeToSmallcaps = ({
         // All other strings pass through to JSON
         return passable;
       }
-      case 'symbol': {
-        // At the smallcaps level, we prefix symbol names with `%`.
-        // By "symbol name", we mean according to `nameForPassableSymbol`
-        // which does some further escaping. See comment on that function.
```

Comment:
Is this comment worth preserving?

Reply:
I don't think so, because it is covered by the package comment and the direct use of `nameForPassableSymbol` in the same way that e.g. 'tagged' doesn't re-explain use of `{ '#tag': …, payload: … }`.

Reply:
Thanks. Resolving.